src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -42,6 +43,10 @@
   <ItemGroup>
       <!--This is used for Source Link -->
        <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All"/>
+      <None Include="$(MSBuildThisFileDirectory)/../docs/assets/images/nugeticon.png">
```

Comment:
Why not use kebab-case for the file name like we do for such files? Also, I don't think this belongs into the docs, this is not docs.

Reply:
Ok, I just followed naming as per favicon

Reply:
You didn't address "Also, I don't think this belongs into the docs, this is not docs."

Reply:
This is a file for Nuget so it should be part of the solution I guess. Else, if we add this to the documentation and we replace the documentation or move it then the solution will fail to resolve this file. So, I agree.

Reply:
Move it to the Solution Items folder in Visual Studio and move the file to the root folder of the source code.

Reply:
Or in the OrchardCore.Build folder.

Reply:
src\OrchardCore.Build would be the best place indeed.

Reply:
Ok, I don't mind.

Reply:
IMO artwork, images related to repo should stay as single location (as source of truth) and should not be duplicated at different places as it adds more confusion

Reply:
I don't disagree with the single location aspect, but this file is still not part of the documentation, rather part of the build inputs. Same reason why we have favicon files in projects and not under docs (but they could actually copy a common file from the build folder too). (BTW there is [a doc page about branding](https://docs.orchardcore.net/en/latest/docs/resources/branding/) but it keeps its assets in a different folder; docs/assets are assets for the documentation website.)

Reply:
I don’t mind moving to build, but it’s obvious that if `favicon` used in src is in `docs/assets`  then nuget icon is also going along within same folder

Reply:
I was referring to favicons in theme projects, for example. As I'm saying, the docs/assets folder is for assets of the documentation website. It's a different app altogether, it could be in a different repository as well, it might be moved around, as it has been, because it's a self-contained project. It has no connection to the projects where the NuGet logo is used.

Reply:
Well, if we keep it also in the documentation it doesn't really make sense because we should only keep the original logos in the documentation normally. And we don't require to add any documentation about Nuget packages. Else, we create documentation about Orchard Core Nuget packages but it is only useful for devs. This means somehow that it should be kept in the OC source code. We can keep it in both places but then create documentation about it in the docs. Else, this is not a file meant for use in the documentation ... you see what I mean there. And yes, if we use it in both places we need to have it in both places because the documentation could be moved around.

Reply:
But here, what is mandatory for now is to move it to the source code like proposed.

Reply:
moved to OrchardCore.Build folder